### ISSUE TITLE
[0.9.0.0 beta staging-1742] Laws : Possibility of griefing / workarounds

### ISSUE THREAD

**Issue by IcedForge** [StrangeLoopGames/EcoIssues#17983]
Right now -
Capitol decides who can do stuff to legal objects.

Scenario - 

Group 1 makes a capitol it gets passed and allows them to setup laws etc 
They make a law that is passed, they place that court inside their claims; removes everyone auth on this law.
Now this law cannot be changed by anyone else regardless if they overwrite the others permissions by overthrowing with a new capitol?
-
Would it be possible to add something that always auths the group / individual that has capitol access with any legal objects regardless of auth?

Is there any possibility to tie in the laws with the constitutions so it would nullify / disable old laws; and or tie then into the plot claim? so unless the capitol is on the same plot of land as the courts the courts would not function or something in this line?

Granted a lot of this is also down to player decisions, law making and interactions but at the same time it also falls borderline on exploitation / clever use of game mechanics in a negative way that should probably be looked into for further updates.

**Comment by NDKilla** [comment]
I'm pretty sure the new government would be able to make a law allowing them to ignore auth and remove the old court and/or remove the law on said court.

The obvious exception would be laws preventing you from changing laws at all, but I'm pretty sure that's not possible.

The problem with the 'automatic auth' you mention is the fact that, by default, nobody with that access is hard coded. Elected titles have to be created before hand, otherwise everything just defaults to being voted on by public and no group actually has any authority whatsoever unless you create that group and laws relating to it.

**Comment by IcedForge** [comment]
Yes but what i mean is that you camn setup laws in a manner that even if you ratify the constitution you cannot vote to remove existing laws as you have no access to the actual objects

**Comment by IcedForge** [comment]
For example, group 1 joins day 1
Pushes o get gov stuff, makes a registrar and setup a group
Use that group as excecutive in the constitution and votes to ratify, they now setup a court with a law stating 
Open inventory object court, If Not user group X then prevent.
Now they have made it so ONLY their group can even access the laws so no one else has a right to even if their constitution is removed as the law would still be in effect if a new one is replaced

**Comment by IcedForge** [comment]
And on further analyzing and some more thought, they can easily just add 
Trigger open inventory capitol/amendments - IF not group 1 then prevent, which would completely block off access to even contesting the government at all.

**Comment by NDKilla** [comment]
Ahh. I see what you mean. 

> Open inventory object court, If Not user group X then prevent.

This would apply to the new courts that would allow them to bypass previous restrictions.

If that works then I think this would definitely be a bug and be correct. I thought I remember reading that you basically weren't supposed to be able to do stuff like this, but I'm not sure if it actually works or not. :D I was just going by previous things I remember hearing.
